README.md

```diff
@@ -13,9 +8,15 @@ and [RDS Enhanced Monitoring via CloudWatch Logs](https://docs.aws.amazon.com/Am
 Based on [Technofy/cloudwatch_exporter](https://github.com/Technofy/cloudwatch_exporter),
 but very little of the original code remained.
```

Comment:
superfluous. Just mention that it bases on Technofy/cloudwatch_exporter or that you forked it

Reply:
Fixed in https://github.com/pdffillerdocker/rds_exporter/commit/4e36931a0807b3e836b4f63cea3352a3ee36ed68